cmd/csi_driver/main.go

```diff
@@ -42,13 +55,203 @@ var (
 	version = "unknown"
 )
 
+func getGvnicNames() ([]string, error) {
+	links, err := netlink.LinkList()
+	if err != nil {
+		return nil, fmt.Errorf("failed to list network interfaces: %w", err)
+	}
+
+	var ethNics []string
+	for _, link := range links {
+		if strings.HasPrefix(link.Attrs().Name, "eth") {
+			ethNics = append(ethNics, link.Attrs().Name)
+		}
+	}
+
+	return ethNics, nil
+}
+
+func configureRoutesForNic(nicName string, ipAddr net.IP, tableID int) error {
```

Comment:
This routing logic seems out of scope for the kmod installer. To keep this PR focused, can we defer these functions to a follow-up PR?